src/list.json

```diff
@@ -141,9 +141,16 @@
         "organization": "Percona",
         "org_url": "https://www.percona.com/",
         "tags": ["Tshirt","Hat"],
-        "description": "This year Percona participates in this fabulous event with its open source software. You can contribute to 12 repositories with Percona Monitoring and Management (PMM), Percona Kubernetes Operators, pg_stat_monitor, and other projects! In addition to official Hacktoberfest prizes, you can receive a piece of Percona swag: a T-shirt or a hat."
+        "description": "This year Percona participates in this fabulous event with its open source software. You can contribute to 12 repositories with Percona Monitoring and Management (PMM), Percona Kubernetes Operators, pg_stat_monitor, and other projects! In addition to official Hacktoberfest prizes, you can receive a piece of Percona swag: a T-shirt or a hat.",
         "no_of_prs": 4,
         "link": "https://www.percona.com/blog/contribute-to-open-source-with-percona-and-hacktoberfest/" 
+      },
+      {
```

Comment:
Then please run this command to Re-style it. **Make sure you are on the repo directory**
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2334274/patch | git am`
`git push`

Reply:
did all that. Had to create a new branch otherwise, restyle won't work. I created a new branch on my forked repo, add successfully ran all the commands you listed. Should I submit a new PR? I'm sorry, new to GitHub, trying to learn.

Reply:
> @thefool76 did all that. Had to create a new branch otherwise, restyle won't work. I created a new branch on my forked repo, and successfully ran all the commands you listed. Should I submit a new PR? I'm sorry, new to GitHub, trying to learn.

Ok then Submit a new one if the checks fails for for eg: restyled  check. Then Please Click on the details and follow the instruction mention there. 
I close this PR then.